docs/SUMMARY.md

```diff
@@ -4,24 +4,27 @@
 
 ## Features
 
-* [🔥🌶️ Bitcoin Native Swaps 🌶️🔥](product-features/bitcoin-swaps/README.md)
+* [🪙 Bitcoin Native Swap](product-features/bitcoin-swaps/README.md)
+  * [💡 Key Concepts](product-features/bitcoin-swaps/key-concepts.md)
+  * [🧑‍🏫 How to Swap](product-features/bitcoin-swaps/how-to.md)
+  * [❓ FAQs](product-features/bitcoin-swaps/faqs.md)
 * [🔄 Token Swaps](product-features/token-swaps/README.md)
-  * [💡 Key concepts](product-features/token-swaps/key-concepts.md)
-  * [🧑‍🏫 How to swap](product-features/token-swaps/how-to.md)
+  * [💡 Key Concepts](product-features/token-swaps/key-concepts.md)
+  * [🧑‍🏫 How to Swap](product-features/token-swaps/how-to.md)
   * [❓ FAQs](product-features/token-swaps/faqs.md)
 * [🐋 Liquidity Pools](product-features/liquidity-pools/README.md)
-  * [💡 Key concepts](product-features/liquidity-pools/key-concepts.md)
-  * [🧑‍🏫 How to add/remove liquidity](product-features/liquidity-pools/how-to.md)
+  * [💡 Key Concepts](product-features/liquidity-pools/key-concepts.md)
+  * [🧑‍🏫 How to Add/Remove Liquidity](product-features/liquidity-pools/how-to.md)
   * [❓ FAQs](product-features/liquidity-pools/faqs.md)
   * [📝 Self-Service Listing](product-features/self-service-listing.md)
 * [🌾 Farming](product-features/farming/README.md)
-  * [💡 Key concepts](product-features/farming/key-concepts.md)
+  * [💡 Key Concepts](product-features/farming/key-concepts.md)
   * [🧑‍🏫 How to farm & harvest](product-features/farming/how-to.md)
   * [❓ FAQs](product-features/farming/faqs.md)
   * [🧑‍🌾 Self-Service Farming](product-features/self-service-farming.md)
 * [⚓ Stake](product-features/staking/README.md)
-  * [💡 Key concepts](product-features/staking/key-concepts.md)
-  * [🧑‍🏫 How to stake](product-features/staking/how-to.md)
+  * [💡 Key Concepts](product-features/staking/key-concepts.md)
+  * [🧑‍🏫 How to Stake](product-features/staking/how-to.md)
```

Comment:
I woldn't change the sub-pages titles to capitalized. I would keep with regular casing, like: How to stake, Key concepts, etc.

Reply:
(in all titles)